inc/settings.php

```diff
@@ -153,6 +153,10 @@ function siteorigin_north_settings_init(){
 		'responsive' => array(
 			'title' => __('Responsive', 'siteorigin-north'),
 			'fields' => array(
+				'disable' => array(
```

Comment:
Very minor, but to keep with convention, this setting should be called `disabled`. The setting names should imply state, rather than action. This means setting names should ideally be adjectives (like disabled), rather than verbs (like disable). Hope that makes sense, but feel free to ask if it doesn't.